.github/workflows/icfy-stats.yml

```diff
@@ -0,0 +1,44 @@
+name: Calculate ICFY stats
+
+on: [push]
+   
+jobs:
+  build:
+    name: Build ICFY stats
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '>=12.15.0'
+    - name: Checkout code
+      uses: actions/checkout@master
```

Comment:
That error looks like we'll need to do a deeper fetch here. Looks like it defaults to shallow:

https://github.com/actions/checkout/blob/f858c22e963bc60bc9d01c3d105c52a45a7deb6e/action.yml#L27-L29